ansible/roles/openshift_clam_update/files/openshift_clam_update.yml

```diff
@@ -0,0 +1,181 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: oso-clam-update
+objects:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      template: oso-clam-update
+    name: "oso-${PLAT}-ops-base"
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: "oso-${PLAT}-ops-base:latest"
+    postCommit: {}
+    resources: {}
+    source:
+      contextDir: "docker/oso-ops-base/${PLAT}"
+      git:
+        ref: clam_update_container
+        uri: https://github.com/rhdedgar/openshift-tools
+      secrets: []
+      type: Git
+    strategy:
+      dockerStrategy:
+        env:
+        - name: OO_PAUSE_ON_BUILD
+          value: "false"
+        from:
+          kind: ImageStreamTag
+          name: "${PLAT}:latest"
+      type: Docker
+    triggers:
+    - imageChange: {}
+      type: ImageChange
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      template: oso-clam-update
+    name: oso-clam-update
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: "oso-${PLAT}-clam-update:latest"
+    postCommit: {}
+    resources: {}
+    source:
+      contextDir: "docker/oso-clam-update/${PLAT}"
+      git:
+        ref: clam_update_container
+        uri: https://github.com/rhdedgar/openshift-tools
```

Comment:
Fix this.  Should come from the /openshift/ repo.

Reply:
Updated.